processing/tagger.py

```diff
@@ -0,0 +1,97 @@
+from operator import itemgetter
+from itertools import groupby
+
+DEFAULT_CONFIDENCE = 0.85
+FALSE_POSITIVE = "false-positive"
+UNIDENTIFIED = "unidentified"
+MULTIPLE = "multiple animals"
+STATUS = "status"
+CLARITY = "clarity"
+
+MESSAGE = 'message'
+CONFIDENCE = 'confidence'
+FALSE_POSITIVE_TAG = {"event": "false positive", CONFIDENCE: DEFAULT_CONFIDENCE}
+
+
+def calculate_tags(tracks, conf):
+  # No tracks found so tag as FALSE_POSITIVE
+  tags = {}
+  if not tracks:
+    tags[FALSE_POSITIVE] = FALSE_POSITIVE_TAG
+    return tags
+
+  toBeTagged, unknowns = findSignificantTracks(tracks, conf)
+
+  # definites
+  sortedTags = sorted(toBeTagged, key=itemgetter("label"))
+  for label, label_tracks in groupby(toBeTagged, itemgetter("label")):
+    confidence = max(t[CONFIDENCE] for t in label_tracks)
+    tags[label] = {CONFIDENCE: confidence}
+
+  # unknowns
+  for track in unknowns:
+    if track[CLARITY] > .5 * conf.min_tag_clarity / 2 and track["label"] in tags:
```

Comment:
The reason for the multiply by 0.5 and divide by 2 aren't clear (why not divide by 4?). Please add a comment to explain what's happening here.